integration_tests/tests/common/mod.rs

```diff
@@ -29,12 +29,11 @@ pub fn test_task_builders(
 ) -> (HpkePrivateKey, TaskBuilder, TaskBuilder) {
     let endpoint_random_value = hex::encode(random::<[u8; 4]>());
     let collector_keypair = generate_test_hpke_config_and_private_key();
-    let leader_task = TaskBuilder::new(QueryType::TimeInterval, vdaf, Role::Leader)
+    let leader_task = TaskBuilder::new(query_type, vdaf, Role::Leader)
         .with_aggregator_endpoints(Vec::from([
             Url::parse(&format!("http://leader-{endpoint_random_value}:8080/")).unwrap(),
             Url::parse(&format!("http://helper-{endpoint_random_value}:8080/")).unwrap(),
         ]))
-        .with_query_type(query_type)
```

Comment:
It seems like we should either have an argument in `TaskBuilder::new` _or_ a `with_whatever` method, but not both.

Reply:
The idea is that `new` takes required parameters for which there is no sensible default. Everything else gets a sensible default and a `with_foo` method to allow the default to be overridden. Query type is more the former, so it goes into the `new` "constructor" method.